website/common/script/libs/shops.js

```diff
@@ -251,6 +257,7 @@ shops.getTimeTravelersCategories = function getTimeTravelersCategories (user, la
               notes: '',
               locked: false,
               currency: 'hourglasses',
+              pinKey: 'TODO',
```

Comment:
Are mounts and pets pinnable? I think you can only pin gem or gold purchasable items